test/e2e/batch/batch_test.go

```diff
@@ -40,7 +40,8 @@ func TestBatchInlineConfig(t *testing.T) {
 	t.Parallel()
 	name := strcase.ToKebab(t.Name())
 	infinispan := createCluster(name)
-	defer testKube.DeleteInfinispan(infinispan, tutils.SinglePodTimeout)
+	defer testKube.CleanNamespaceAndLogOnPanic(tutils.Namespace, infinispan.Labels)
```

Comment:
@durgeshanaokar you've missed to set `map[string]string{"test-name": t.Name()}` for all Infinispan CR's in the Batch tests

Reply:
@dmvolod in batch tests all the the CRs are created by  createCluster and createBatch, I set the labels in these methods. C